### ISSUE TITLE
License

### ISSUE THREAD

**Issue by andjoer** [dapascual/K2T#5]
Hi, 
thanks for the clean and easy to read implementation! I would like to use and modify the code and add custom conditions for deleting branches, masking logits etc. 
As far as I can see there is not any license applied to your codes, so officially I  would not be allowed to do any of these and my university is strict in this concern. Would I be allowed to do above and present our work - of course with proper citation?  It would be a building brick in a larger model.

**Comment by dapascual** [comment]
Hey, sorry for the late answer, I somehow missed this issue. Sure, as long as you cite it it's not a problem.
